test/CMakeLists.txt

```diff
@@ -442,7 +444,7 @@ endforeach()
 # iodaconv_comp.sh test.
 #
 # For a converter that is generating new data, use a non-zero tolerance.
-set(IODA_CONV_COMP_TOL_ZERO "0.0")
+set(IODA_CONV_COMP_TOL_ZERO "1.0e-4")
```

Comment:
If you set this below IODA_CONV_COMP_TOL  below does it still pass?

Reply:
@PatNichols Yes, it still passes. Is that the preferred order? If so, I'll adjust the `amsr2` converter as well.

Reply:
@smaticka if still passes then use the smaller value.